app/models/manageiq/providers/openstack/network_manager/cloud_subnet.rb

```diff
@@ -1,5 +1,13 @@
 class ManageIQ::Providers::Openstack::NetworkManager::CloudSubnet < ::CloudSubnet
-  def self.raw_create_subnet(ext_management_system, options)
+  include ProviderObjectMixin
+  include AsyncDeleteMixin
+  include SupportsFeatureMixin
+
+  supports :create_cloud_subnet
```

Comment:
There has been some confusion around that one and different answers, so I leave it for now.